addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -743,6 +747,19 @@ export const editorCommands = {
         row.remove();
         siblingRow ? setSelection(...startPos(siblingRow)) : editorCommands.deleteTable(editor, table);
     },
+    resetSize: (editor,table) => {
+        if (!table) {
+            getDeepRange(editor.editable, { select: true });
+            table = getInSelection(editor.document,'table');
+            if (!table) return;
+        }
+        table && table.removeAttribute('style');
```

Comment:
If you don't have `table` you `return` higher up in the code so you don't need to check it again here.